### DISCUSSION TITLE
How to access flow webhook response?

### DISCUSSION THREAD

**Discussion by theknigge** [directus/directus#16873] [Archive]
Hello, maybe it's a stupid question (I'm a newbie by Directus), but how can I access an webhook response in another operation?
Described [in the docs](https://docs.directus.io/configuration/flows/operations.html#webhook-request-url):
> When an operation completes successfully, the response is appended under its operationKey.

So I thought that I can access it by $trigger.payload._MyOperationKey_, but the result is always "undefined". 
Also tried $trigger.payload.MyOperationKey.response

Webhook is working normally/without errors.

**Comment by br41nslug** [comment]
The operationKey is the first part 😄  `<operationKey>.some.property` you can think of `$last` and `$trigger` as shortcut operationKey so you don't have to remember all the keys of operations in your flows.

**Reply by theknigge** [reply]
Thank you 💯
